MVCapplication/src/main/webapp/WEB-INF/views/login.jsp

```diff
@@ -0,0 +1,47 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core"%>
+<%@ taglib uri="http://tiles.apache.org/tags-tiles" prefix="tiles"%>
+<%@ taglib prefix="form" uri="http://www.springframework.org/tags/form"%>
+<%@ page isELIgnored="false" %>
+
+<div class="container">
+    <div class="row">
+        <div class="col-md-4 col-md-offset-4">
+            <div class="login-panel panel panel-default">
+                <div class="panel-heading">
+                    <h3 class="panel-title">Please Sign In</h3>
+                </div>
+                <div class="panel-body">
+                    <form:form role="form" method="POST" modelAttribute="loginForm">
+                        <fieldset>
+                            <c:if test="${not empty loginError}">
+                                <div class="alert alert-danger">
+                                    <span class="glyphicon glyphicon-minus-sign" aria-hidden="true"></span>
+                                    <c:out value="${loginError}" />
```

Comment:
c:out is not required unless you need to display html tags. You can simply use EL ${loginError}